web/src/engine/websites/ThreeHentai.ts

```diff
@@ -0,0 +1,21 @@
+import { Tags } from '../Tags';
+import icon from './ThreeHentai.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { ExtractImageLink } from './NHentai';
```

Comment:
Are these sites related, so it i safe to assume they share the same business logic?
Otherwise changing this method may unintentionally break this website in the future.